src/detect-jsx.js

```diff
@@ -0,0 +1,19 @@
+const { parse } = require("@babel/parser");
+const traverse = require("../babel-traverse/lib/index.js").default;
+
+const { parseOptions } = require("./convert.js");
+
+const detectJsx = tsCode => {
```

Comment:
`detectJsx` is called with flow code.  Eventually I'd like this project to do conversions both ways so I think renaming `tsCode` to `code` is appropriate.